liquifier.js

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const yargs = require(`yargs`);
+const hideBin = require(`yargs/helpers`).hideBin;
+
+/**
+ * @typedef {Object} Contract~AbiOption~Input
+ * @property {string} internalType
+ * @property {string} name
+ * @property {string} type
+ * @property {boolean} [indexed]
+ * @property {boolean} [optional]
+ * @property {*} [defaultValue]
+ */
+
+/**
+ * @typedef {Object} Contract~AbiOption
+ * @property {string} type
+ * @property {string} name
+ * @property {Contract~AbiOption~Input[]} inputs
+ * @property {Contract~AbiOption~Input[]} [outputs]
+ * @property {boolean} [anonymous]
+ * @property {string} [stateMutability]
+ */
+
+/**
+ * @typedef Contract
+ * @property {string} contractName
+ * @property {Contract~AbiOption[]} abi
+ */
+
+// **
+
+const arguments = yargs(hideBin(process.argv))
+  .usage(`Usage: $0 [options]`)
+  .alias(`f`, `file`)
+  .nargs(`f`, 1)
+  .describe(`f`, `File to parse`)
+  .alias(`d`, `dir`)
+  .default(`d`, `./`)
+  .describe(`d`, `Output dir`)
+  .alias(`i`, `interface-dir`)
+  .describe(`i`, `Folder to create the interface file on`)
+  .default(`i`, `./src/interfaces/`)
+  .alias(`o`, `override`)
+  .describe(`o`, `Override existing output file on -d`)
+  .boolean(`o`)
+  .default(`o`, false)
+  .alias(`s`, `showoff`)
+  .describe(`s`, `Dry run`)
+  .boolean(`s`)
+  .default(`s`, false)
+  .alias(`S`, `SHOWOFF`)
+  .describe(`S`, `Dry run with show source at the end`)
+  .boolean(`S`)
+  .default(`S`, false)
+  .alias(`v`, `verbose`)
+  .describe(`v`, `Blablabla blabla, bla blaa!`)
+  .boolean(`v`)
+  .default(`v`, false)
+  .demandOption([`f`,])
+  .help(`h`)
+  .alias(`h`, `help`)
+  // .epilog('copyright bepro')
+  .argv;
+
+// **
```

Comment:
Unused line

Reply:
Thanks for the help @riaje, great to have you here! :)